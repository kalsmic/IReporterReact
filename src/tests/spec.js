import React from 'react';
import { shallow } from 'enzyme';
import IReporterApp from '../components/App';


describe('<IReporterApp Component', () => {
  const component = shallow(<IReporterApp />);

  it('should render without crashing', () => {
    expect(component).toMatchSnapshot();
  });
});
