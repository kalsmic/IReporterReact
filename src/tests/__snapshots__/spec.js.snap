// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IReporterApp Component should render without crashing 1`] = `
<BrowserRouter
  forceRefresh={true}
>
  <Switch>
    <OnlyNoneAuthenticatedRoute
      component={[Function]}
      exact={true}
      path="/"
      strict={true}
    />
    <OnlyNoneAuthenticatedRoute
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(SignUp)",
          "type": [Function],
        }
      }
      exact={true}
      path="/signup"
      strict={true}
    />
    <AuthenticatedRoute
      component={[Function]}
      exact={true}
      path="/createIncident"
      strict={true}
    />
    <AuthenticatedRoute
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(ViewIncident)",
          "type": [Function],
        }
      }
      exact={true}
      path="/:incidentType/:incidentUUID"
      strict={true}
    />
  </Switch>
</BrowserRouter>
`;
